lang/cs/Org.Apache.REEF.Wake.Tests/StreamingTransportTest.cs

```diff
@@ -168,5 +189,16 @@ private static ITcpPortProvider GetTcpProvider(int portRangeStart, int portRange
                 .Build();
             return TangFactory.GetTang().NewInjector(configuration).GetInstance<ITcpPortProvider>();
         }
+
+        private static ITcpClientFactory GetTcpClientFactory(int connectionRetryCount, int sleepTimeInMs)
+        {
+            var config =
```

Comment:
I cannot use it here since it involves providing Driver configursations and all and Wake and hence its tests are independent of it.